### ISSUE TITLE
Circuit python compatibility

### ISSUE THREAD

**Issue by jtbuckland** [sb-ocr/ocreeb-12#10]
I found there was an issue with the latest version of CircuitPython having an error when loading the board I noticed in the video version 7.3.3 was used which appears to have fixed the issue.

**Comment by R8bert** [comment]
Yes. New versions cause errors, maybe they can be solved but right now i just wanted to check if my solder skills are not as bad as I expected. This comment really saved me, thanks!
